strawberryfield.module

```diff
@@ -189,120 +158,97 @@ function strawberryfield_invalidate_fieldefinition_caches(ContentEntityInterface
- *
- * @param \Drupal\Core\Entity\EntityTypeInterface $entity_type
- *
- * @return array
+ * Implements hook_entity_base_field_info().
  */
 function strawberryfield_entity_base_field_info(EntityTypeInterface $entity_type) {
-  if ($entity_type->id() == 'node') {
-    $scheme_options = \Drupal::service('stream_wrapper_manager')->getNames(
-      StreamWrapperInterface::WRITE_VISIBLE
-    );
-    if (isset($scheme_options['private'])) {
-      $schema = 'private';
-    }
-    elseif (isset($scheme_options['public'])) {
-      $schema = 'public';
-    }
-    else {
-      $schema = 'public';
-    }
-
-    $fields = [];
-    // Add a field that serves as a drop box for any entities that bear a SBF
-    // @see https://www.drupal.org/project/drupal/issues/2346347
-    // (still WIP as June 2020)
-    // know why we can't use \Drupal\Core\Field\BaseFieldDefinition
-    // @TODO If we try to make this Bundle specific?
-    // @Update 20202: https://www.previousnext.com.au/blog/how-create-and-expose-computed-properties-rest-api-drupal-8
-    // Issue with that approach is we need to have a hook update for every bundle
-    // Which makes adding new bundles and attaching automagically super complex
-
-    // @TODO future work on exposing other JSON properties as other REAL field types
-    // Dynamically can be achieved by creating a new Class extending BaseFieldDefinition
-    // That manages without hickups the 'Bundle' option
-    // \Drupal\Core\Field\BaseFieldDefinition::setTargetBundle
-
-    $fields['field_file_drop'] = BaseFieldDefinition::create('entity_reference')
-      ->setName('field_file_drop')
-      ->setLabel(t('Drop Files'))
-      ->setComputed(TRUE)
-      ->setRevisionable(FALSE)
-      ->setTranslatable(FALSE)
-      ->setCardinality(FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED)
-      ->setReadOnly(FALSE)
-      ->setTargetEntityTypeId($entity_type->id())
-      ->setSettings(
-        [
-          'target_type' => 'file',
-          'file_directory' => 'sbf_tmp',
-          'uri_scheme' => $schema,
-        ]
-      )
-      ->setClass(StrawberryFieldFileComputedItemList::class)
-      ->setDisplayConfigurable('view', FALSE)
-      ->setDisplayConfigurable('form', FALSE);
-
-
-    // A computed field to store customly detected Node references from a SBF.
-    $fields['field_sbf_nodetonode'] = BaseFieldDefinition::create('entity_reference')
-      ->setName('field_sbf_nodetonode')
-      ->setLabel('Related ADOs')
-      ->setSetting('target_type', 'node')
-      ->setTargetEntityTypeId('node')
-      ->setDescription(t('Computed Node to Node relationships'))
-      ->setComputed(TRUE)
-      ->setCardinality(FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED)
-      ->setRevisionable(FALSE)
-      ->setReadOnly(TRUE)
-      ->setTranslatable(FALSE)
-      ->setClass(StrawberryFieldEntityComputedItemList::class);
-    return $fields;
+  if ($entity_type->id() != 'node') {
```

Comment:
This are Indents. Idea for the future: What if we make, in pulls, that will bring large indent/DCS separate commit? so we do not need to check if something other than indents changed and introduce errors when not? This applies to me. I'm fine and happy we (you) are doing a lot of needed DCS changes. I just want to be sure I can separate new code/ old code when doing a review an testing

Reply:
I am a big fan of leaving the field greener as we go, otherwise will never ever get done (my experience at least) and will endlessly conflict with other changes.

Reply:
I agree and this is our way too. My only though is that a single pull can have many commits, so we can bring DCS changes as a single commit together. Again, I can start doing that and you can see if it works for you too when reviewing my code.

Reply:
If entities implement `EntityPublishedTrait` without implementing the right interface they need to fix :) IMHO